Source/WebKit/UIProcess/Cocoa/NavigationState.mm

```diff
@@ -1541,7 +1541,7 @@ static _WKSOAuthorizationLoadPolicy wkSOAuthorizationLoadPolicy(SOAuthorizationL
         }
         if (!m_networkActivity) {
             RELEASE_LOG(ProcessSuspension, "%p - NavigationState is taking a process network assertion because a page load started", this);
-            m_networkActivity = webView->_page->legacyMainFrameProcess().throttler().backgroundActivity("Page Load"_s).moveToUniquePtr();
+            m_networkActivity = webView->_page->legacyMainFrameProcess().throttler().foregroundActivity("Page Load"_s).moveToUniquePtr();
```

Comment:
This is controversial I think. If you start a load in a visible WKWebView and background the app, do you really still expect the WebProcess to have a *foreground* assertion for up to 30 seconds? This has power implications.

Reply:
Good point! I'll remove this change from the patch.